src/entities/protocols/uniswap.ts

```diff
@@ -45,7 +45,7 @@ export class UniswapTrade implements Command {
     if (this.trade.inputAmount.currency.isNative) {
       // TODO: optimize if only one v2 pool we can directly send this to the pool
       planner.addCommand(CommandType.WRAP_ETH, [
-        ADDRESS_THIS,
+        ROUTER_AS_RECIPIENT,
```

Comment:
woops I think I forgot about this. FYI changed this since with the new changes, this is now an external var needed for integration so wanted to give it more color